src/main/kotlin/world/anhgelus/msmp/basicmazeworldgenerator/events/PlayerListener.kt

```diff
@@ -86,4 +86,18 @@ object PlayerListener: Listener {
         Bukkit.dispatchCommand(Bukkit.getConsoleSender(), "datapack enable \"file/basicmazeworldgenerator\"")
         dpEnabled = true
     }
+
+    @EventHandler
+    fun onMove(event: PlayerMoveEvent) {
```

Comment:
Maybe switch to schedule task because this function can be called very often and cause lags + bug with Minecart or llamas

Reply:
If this code is working, I'll definitely improve this code with a schedule task